server/common/common-service/src/main/java/io/holoinsight/server/common/security/InternalWebApiInterceptor.java

```diff
@@ -31,7 +28,7 @@
 public class InternalWebApiInterceptor implements HandlerInterceptor {
   static final Set<String> LOCALHOST = Sets.newHashSet("localhost", "127.0.0.1", "0:0:0:0:0:0:0:1");
   private static final Set<String> INTERNAL_WHITE_HOSTS =
-      Sets.newHashSet("gateway.holoinsight-gateway", "gateway.holoinsight-server");
+      Sets.newHashSet("gateway.holoinsight-gateway", "gateway.holoinsight-server", "server");
```

Comment:
Don't add 'server' here.
Add holoinsight.security.whiteHosts properties in application.yaml